### ISSUE TITLE
Test set of CT2Rep

### ISSUE THREAD

**Issue by sunnyyy17** [ibrahimethemhamamci/CT2Rep#12]
I would like to ask is the test set you have used for your related work CT2Rep the same as the validation set provided here in CT-RATE? I would like to make some comparision with my reproduction of your work CT2Rep. 

Thank you very much.

**Comment by sunnyyy17** [comment]
Dear authors, 

I am sorry to disturb you but may I ask if I may hear from you about my question on the test set of CT2Rep? I am asking because I wish to compare my work to yours but I am wondering if I may compare it directly to the result you have shown on your paper or whether for correct comparision I should run my own evaluation on the validation sets of CT-RATE after I re-implement CT2Rep on my computing environment.  

Thank you very much :)

**Comment by hari3100** [comment]
> Dear authors,
> 
> I am sorry to disturb you but may I ask if I may hear from you about my question on the test set of CT2Rep? I am asking because I wish to compare my work to yours but I am wondering if I may compare it directly to the result you have shown on your paper or whether for correct comparison I should run my own evaluation on the validation sets of CT-RATE after I re-implement CT2Rep on my computing environment.
> 
> Thank you very much :)

Hi @sunnyyy17 , I hope you are doing well, I'm commenting in hopes that you have already ran the CT2Rep model on your own system, could you help me with the inference of this model i have a few questions as well regarding the input size, etc. 
But the main thing I'm looking for is how can I run the model, I'm new to medical imaging ,and I'm not understanding the code which is already present in the repo, could you maybe share some inference code you wrote, or guide me on how to do it ?....any feedback or guidance will be greatly appreciated...Thank you for your time!

**Comment by sunnyyy17** [comment]
Hello @hari3100 , I haven't explicitly written a separate inference code for this model but you can take a look at the evaluation code inside the Trainer() class inside trainer.py around line 198 to line 210 for further details in how to load the image and report for correct inference! Thank you!

**Comment by sunnyyy17** [comment]
Dear authors, 

May I ask if I may hear from you on my question on whether the validation set of CT-RATE is identical to the test set you have used for CT2Rep?  

Hope you are doing well :)

Thank you very much
